plugin/trino-exchange/src/main/java/io/trino/plugin/exchange/FileSystemExchange.java

```diff
@@ -225,9 +235,9 @@ private Path getExchangeDirectory()
     @Override
     public ExchangeSourceSplitter split(ExchangeSourceHandle handle, long targetSizeInBytes)
     {
-        // always split for testing
-        LocalFileSystemExchangeSourceHandle localHandle = (LocalFileSystemExchangeSourceHandle) handle;
-        Iterator<Path> filesIterator = localHandle.getFiles().iterator();
+        // TODO: replace with correct split implementation
```

Comment:
//TODO(link to github issue)

Reply:
Actually I discussed with @arhimondr, we will always split into files for now as we always re-assemble them with respect to `targetSizeInBytes` , so removing this TODO for now. Not sure if it's the best arrangement.